src/infer_response.cc

```diff
@@ -179,6 +198,34 @@ InferResponse::Error()
   return error_;
 }
 
+void
+InferResponse::SetNextResponseFuture(
```

Comment:
Can we delete this function too? It is not used anywhere.

Reply:
This got slipped from my previous changes. Deleted the unused functions and variable.